app/assets/stylesheets/components/_table.scss

```diff
@@ -0,0 +1,20 @@
+table.dataTable {
+  border: 1px solid lighten($light-gray, 25%);
+  box-shadow: none;
+  width: 100% !important;
+
+  thead, tfoot, tr, tr.even { background: white; }
+  tr td,
+  tfoot tr th,
+  thead tr th { color: $light-gray; }
+
+  td:before {
+    background-color: #2CDF79 !important;
```

Comment:
Color literals like `#2CDF79` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#2CDF79` should be written as `#2cdf79`<br>!important should not be used